spec/db/models.rb

```diff
@@ -51,9 +51,9 @@ def nick_or_name
   end
 
   # a (local) virtual_attribute without a uses, but with arel
-  virtual_attribute :nick_or_name, :string do |t|
+  virtual_attribute :nick_or_name, :string, :arel => (lambda do |t|
```

Comment:
Interesting, we were calling `.call` previously... was this a bug previously or something else?

Reply:
this was a bug - it wasn't defining the arel (and caused errors in my new tests)